### PR TITLE
add import_csv and delete_all_dois commands

### PR DESCRIPTION
**Pull request by skasberger**
Addition of two command line functions:
* import_csv: imports the CSV file with the dois. If no filename is passed, it uses a default one.
* delete_all_dois: deletes all entries in the doi table.

### CODE REVIEW

File: app/__init__.py [hunk 1]
```diff
@@ -75,6 +78,58 @@ def create_app():
...
+        """Import CSV file with DOI's.
+
+        Args:
+            filename: filename with path.
+        """
+        import pandas as pd
+        from app.models import Doi
+
+        if not filename:
+            filename = 'app/static/data/PKP_20171220_100.csv'
```
Comment (Bubblbu) [inline app/__init__.py]: this file seems to be missing

File: README.md [hunk 2]
```diff
@@ -83,10 +83,18 @@ Then set the `YOURAPPLICATION_MODE` variable inside your shell, so the right set
...
+export SQLALCHEMY_DATABASE_URI='postgresql://localhost/fhe_collector'
+```
+
 **Start app**
 
 ```
 export FLASK_APP=fhe.py
+export SQLALCHEMY_DATABASE_URI='postgresql://localhost/fhe_collector'
+export YOURAPPLICATION_MODE='DEVELOPMENT'
 flask run
```
Comment (Bubblbu) [inline README.md]: Should I be able to simply execute `flask run` after checking out this PR?

File: README.md [hunk 3]
```diff
@@ -83,10 +83,18 @@ Then set the `YOURAPPLICATION_MODE` variable inside your shell, so the right set
 * 'TESTING': to execute the tests.
 * 'PRODUCTION': to run in production mode.
 
+Set your database. The following example show how to connect to your postgreSQL database `fhe_collector`.
+
+```
+export SQLALCHEMY_DATABASE_URI='postgresql://localhost/fhe_collector'
```
Comment (Bubblbu) [inline README.md]: is using env variables the common way to setup the postgres database?

File: app/__init__.py (hunk 1 above)
Comment (skasberger) [reply app/__init__.py]: yes. can i add it to the repo? its from you.

File: README.md (hunk 2 above)
Comment (skasberger) [reply README.md]: yes. with the environment variables set as stated in the lines before.

File: README.md (hunk 3 above)
Comment (skasberger) [reply README.md]: yes. so you can adapt it to your working environment. can also be set inside the setting-files.

**Comment by Bubblbu** [comment]
A few things for this PR:

- Could you add some more documentation around the usage of `flask db migrate` and `flask db upgrade`? I am not familiar with flask/postgresql so a little bit of language around the purpose of the two-liner would be great. When exactly is it required to do these? Also, what does `"COMMENT"` refer to?
- Similar to the previous point, some additional language throughout the deployment would be great. At the moment it is perfectly fine to copy & paste and get the app running. But I am quite lost at this point as there is not more to interact with. Did the deployment succeed? Is the app successfully running and can I access endpoints? Again, documentation for users that aren't familiar with Flask/postgres would be great which at least introduces the basic places I should look at in order to run tests and verify functionality
- I've got the app running. How do I import a CSV? Could you add the required file and point me to the right places to test this.

Cheers,
Asura

**Comment by skasberger** [comment]
I will update the documentation tomorrow. So if everything is running as intended (without the csv file) I would suggest to merge the PR. I will then make a new PR with the new code and the csv file in it.

**Comment by Bubblbu** [comment]
That sounds good. Approved and merged.

**Comment by skasberger** [comment]
great that you merged and pulled together! <3

**Pull request merged**